content/components/overview/EndingRecipeCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactElement } from "react";
+import { Link } from "react-router-dom";
+import { Alert, Icon, IconButton, Panel, Tag } from "rsuite";
+
+import { useSelectedNormandyEnvironmentAPI } from "devtools/contexts/environment";
+import { RecipeV3 } from "devtools/types/recipes";
+
+export const EndingRecipeCard: React.FC<{
```

Comment:
Personally I'd put this and `EndingRecipes` in the same file; they aren't very useful independent of each other. I don't think we have a hard rule about one-component-per-file, though if we do then lets stick with it.